### ISSUE TITLE
[Feature Request] Display builtin shortcuts in Preferences and make them able to be cusmotized

### ISSUE THREAD

**Issue by c02y** [cboxdoerfer/fsearch#243]
I haven't seen any available shortcuts in fsearch Preference. But after I randomly hit some shortcuts I noticed some shortcuts do exist. I searched the GitHub repo issue page and see no one talking about them.

Then after searching the code directly for a while, I noticed 
https://github.com/cboxdoerfer/fsearch/blob/91ab0421f5cf3c899db1f79dc3a9be4299f9ab9c/src/fsearch.c#L552-L574

So there are multiple builtin shortcuts:
~~~
Tab --> toggle_focus
Ctrl-f --> focus_search
Ctrl-n --> new_window
Ctrl-a --> select_all
Escape --> hide_window
Ctrl-i --> match_case
Ctrl-r --> search_mode
Ctrl-u --> search_in_path
Ctrl-Shift-u --> update_database
Ctrl-p --> preferences
Ctrl-q --> quit
~~~

So can you add these shortcuts info into Preference windows? And it would even better if we can customize them.

**Comment by cboxdoerfer** [comment]
Yes, customizable shortcuts would be great and are definitely coming at one point, I'm just really busy at the moment and don't have much time to work on FSearch.

Adding a list with all the current keyboard shortcuts is much less work, so hopefully I can implement that soon.
